PCB/2400MHz/RX_Nano/README.md

```diff
@@ -61,4 +61,7 @@ https://youtu.be/bOKTOFyyGHo
 
 I made the below image as a quick reference to help when placing components.
 
+Pay attention to the crystal's placement. Orientation differs on older PCBs.
+In any case you want the crystal's pin number 1 to be grounded. 
```

Comment:
Oh, I just checked the datasheet, pin 1 is marked as xtal, it's pins 2 and 4 that are ground

Reply:
Good catch will fix.